### PR TITLE
[_]: Feat/fallback permission

### PR DESCRIPTION
**Pull request by miguelsw**
Issue:
- fallback was running when items were syncing 

update:
- add permission to avoid running the fallback when something in syncing

### CODE REVIEW

File: src/apps/main/remote-sync/handlers.ts [hunk 1]
```diff
@@ -120,16 +120,18 @@ eventBus.on('USER_LOGGED_OUT', () => {
...
+  event.sender.send('CHECK_SYNC_ENGINE_RESPONSE', '');
 });
 
 ipcMain.on('CHECK_SYNC_CHANGE_STATUS', async (_, placeholderStates) => {
-  await sleep(2_000);
-  Logger.info('[SYNC ENGINE] Changing status');
-  remoteSyncManager.placeholderStatus = 'SYNCING';
-  await sleep(7_00);
+  Logger.info('[SYNC ENGINE] Changing status', placeholderStates);
+  await sleep(5_00);
```
Comment (JoanVicens) [inline src/apps/main/remote-sync/handlers.ts:128]: 5_00 or 5_000?

Comment (miguelsw) [reply src/apps/main/remote-sync/handlers.ts:128]: yeah 5_000

File: src/apps/main/remote-sync/RemoteSyncManager.ts [hunk 2]
```diff
@@ -50,6 +53,9 @@ export class RemoteSyncManager {
   getSyncStatus(): RemoteSyncStatus {
     return this.status;
   }
+  getLastSyncingFinishedTimestamp() {
```
Comment (JoanVicens) [inline src/apps/main/remote-sync/RemoteSyncManager.ts:56]: It's only used from the same, this could be private or deleted

**Review by JoanVicens — CHANGES_REQUESTED** [review]
Looks good to me, the only worry I have is the 5_00 is a typo

**Review by JoanVicens — APPROVED** [review]

**Pull request merged**